docs/source/introduction.md

```diff
@@ -42,8 +42,9 @@ You can certainly try to read or write using unsupported drivers that are
 available in your installation, but you may encounter errors.
 
 Note: different drivers have different tolerance for mixed geometry types, e.g.,
-MultiPolygon and Polygon in the same dataset. You will get exceptions if you
-attempt to write mixed geometries to a driver that does not support them.
+MultiPolygon and Polygon in the same dataset. You will get exceptions or 
```

Comment:
Given all the work on this topic in other PRs at the moment, I'd suggest deleting this.